com.super_productivity.SuperProductivity.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.super_productivity.SuperProductivity
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+separate-locales: false
+rename-icon: superproductivity
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  # Use socket=x11 instead of socket=fallback-x11 to make it easier for people to just
+  # unset ELECTRON_OZONE_PLATFORM_HINT if they have problems with Electron's
+  # experimental Wayland support
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  # Allow local file attachments
+  - --filesystem=home:ro
```

Comment:
why does this need full home access?

Reply:
The application has the ability to "attach" any local file to a task item. As far as I could tell, there was no way to allow this without an explicit filesystem mount, but maybe that could be fixed upstream with a different way of opening files.

Reply:
For reference, I believe the app uses [shell.openPath](https://www.electronjs.org/docs/latest/api/shell#shellopenpathpath), which seems to call out to [xdg-open](https://github.com/electron/electron/blob/578c1e5dae1cb341c97d329902abc089b4f7c612/shell/common/platform_util_linux.cc#L342). I don't know that there's currently a more correct way to do this. It seems like opening a file:// URI might do what the user expects.